### DISCUSSION TITLE
Is there a parameterization toolkit available to generate slater koster files ?

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1487] [Q&A]
Hello everyone, 
I am a new user of DFTB+ and I want to use DFTB theory and DFTB+ code in my research project.

As far as I am into the theory of DFTB, all the results of DFTB are slater koster dependent and there are limited slater koster sets available to present the repulsive potential or other Hamiltonian and Overlap matrix elements. 

So, my question is do you also offer any parametrization toolkit that can generate the slater koster files for the pair of species of interest?

for example I want the pair wise interaction between Mg and C to simulate my system but I cant see any parameter set of DFTB level SCC-DFTB, (except DFTB3), which I for now don't know how to use.

**Comment by mona23sarma** [comment]
I am also looking for such a toolkit but unable to find any!

**Comment by bhourahine** [comment]
See https://github.com/dftbplus/skprogs

**Reply by mona23sarma** [reply]
Thank you! I will look into.
